src/tutorial/runtime/introduction.rst

```diff
@@ -81,14 +81,16 @@ We can also call other scripts or executables in this way, e.g:
    Cylc will automatically prepend it to the :envvar:`PATH` environment
    variable when the task runs.
 
+``bin/hello_world``:
+
 .. code-block:: bash
-   :caption: bin/hello_world
```

Comment:
Can leave these captions unchanged.

Reply:
Without the change they are getting numbering e.g. 'Listing N: bin/hello_world' written out above. Just to check, you are aware of that & fine with it in this case?

Reply:
"Listing N" was being added to the captions `numfig` config which this PR has removed.

Reply:
Ah! I did not realise that. Thanks for the pointer. I'll revert the changes to pull out the previously-affected code block captions in a new commit, then, then squash that down.